test/Plot/JobBatcher/JobBatcher.purs

```diff
@@ -0,0 +1,23 @@
+module Test.Expression.JobBatcher
```

Comment:
The namespace of the module seems wrong.

Reply:
Resolved in latest commit